.github/ISSUE_TEMPLATE/documentation.yaml

```diff
@@ -0,0 +1,19 @@
+name: Documentation
```

Comment:
Helm docs live over in https://github.com/helm/helm-www -- is there a way to direct users who want doc improvements over there?